### PR TITLE
Space Travellors' Hub

### PR DESCRIPTION
**Pull request by DilsherB**
In this project we @DilsherB and @Mahamadou-Nouridine did the following:
1. created react app
2. install dependencies
3. created `routs` and `views`
4. NavBar created and `NavLink` applied
5. fetch data from API
6. show data in a proper format on pages `Rockets` and `Missions`
7. Actions and reducers written for both `Rockets` and `Missions`
8. `Reserve` and `Cancel Reserve` functionality applied for `Rockets`
9. `Join Mission` and `Leave Mission` functionality applied for `Missions`
10. conditional rendering applied
11. `My Profie` section created which shows `Reserved Rockets` and `Joined Missions`
12. Readme updated

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import store from "./redux/store";
+import App from "./App";
+
+const root = ReactDOM.createRoot(document.getElementById("root"));
+root.render(
+  <React.StrictMode>
+    <Provider store={store}>
+      <App />
+    </Provider>
+  </React.StrictMode>
```
Comment (TadesseDev) [inline src/index.js:13]: - [ ] Dear Team, Please keep in mind for this milestone you are expected to write unit tests with React Testing Library and Jest (for unit tests). Testing is one major part of this day's development and provides an assertion over your overall project. Please add tests using the `React Testing Library and Jest`

File: src/App.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      path="/"
+      element={<Layout />}
+      errorElement={<div>Error Element</div>}
+      ErrorBoundary={<div>Error Boundaries</div>}
+    >
+      <Route path="/" element={<Rockets />} />
+      <Route path="/missions" element={<Missions />} />
+      <Route path="/myProfile" element={<MyProfile />} />
+    </Route>
+  )
```
Comment (TadesseDev) [inline src/App.jsx:28]: - [ ] It's great that you have a blue text on the active page team ✅. But It's a common practice and provides an advantage when it comes to UX to indicate an active page with underlined text. Please add an underline on the active page.

image

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi Team

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7




## Highlights
 - [x] Good job overall following the template design ✅
 - [x]  Correct GitFlow  ✅
 - [x]  Good commit history  ✅
 - [x]  Good PR title/description  ✅
 - [x]  All linters passed  ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi team, 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Well done adding tests and updating the style for the active link. ✔
- You successfully completed all the requirements.  🎉 🎉 🎉 ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**